dev/io.openliberty.data.internal.persistence/resources/io/openliberty/data/internal/persistence/resources/CWWKDMessages.nlsprops

```diff
@@ -24,5 +24,70 @@
 #   which do NOT contain replacement variables are NOT processed by the MessageFormat class 
 #   (single quote must be coded as one single quote '). 
 
-# All messages must use the range CWWKD1100 to CWWKD1499
+# All messages must use the range CWWKD1000 to CWWKD1399
+
+CWWKD1000.repo.general.err=CWWKD1000E: The {0} method of the {1} repository \
+ interface encountered an error indicating that the repository method might \
```

Comment:
> this message seems oddly vague, I assume because we have no way of knowing what {2} will be, and there might be cases where the method encounters an error, but is still valid?

Yes, that is why. This subtle aspect of this is that in some cases the method itself might be fine and the error could be some other problem.

I can split it into two sentences as you suggest, but would then prefer to word it as,

"The error might indicate that the repository method is not valid."